concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -449,7 +464,67 @@ jobs:
             passed: ['generate-bosh-config']
             trigger: true
           - get: cf-secrets
-
+          - get: metrics-cert
+          - get: vpc-tfstate
+          - get: concourse-tfstate
+          - get: bosh-tfstate
+          - get: vpc-terraform-state
+      - task: vpc-terraform-outputs-to-sh
+        config:
+          platform: linux
+          image: docker:///ruby#2.2-slim
+          inputs:
+          - name: paas-cf
+          - name: vpc-terraform-state
+          outputs:
+          - name: vpc-terraform-state
+            path: terraform-outputs
+          run:
+            path: sh
+            args:
+            - -e
+            - -c
+            - |
+              ruby paas-cf/concourse/scripts/extract_tf_vars_from_terraform_state.rb \
+              < vpc-terraform-state/vpc.tfstate \
+              > terraform-outputs/tfvars.sh
+              ls -l terraform-outputs/tfvars.sh
+              . terraform-outputs/tfvars.sh
```

Comment:
We don't really need to add this `ls` and load the file here. I guess this was for debugging.